storage/mysql/mysql.go

```diff
@@ -24,41 +24,70 @@ import (
 )
 
 const (
-	defaultMaxTuplesInWrite     = 100
-	defaultMaxTypesInDefinition = 100
+	defaultMaxTuplesPerWrite             = 100
+	defaultMaxTypesPerAuthorizationModel = 100
 )
 
 type MySQL struct {
-	db                       *sql.DB
-	tracer                   trace.Tracer
-	logger                   logger.Logger
-	maxTuplesInWrite         int
-	maxTypesInTypeDefinition int
+	db                            *sql.DB
+	tracer                        trace.Tracer
+	logger                        logger.Logger
+	maxTuplesPerWrite             int
+	maxTypesPerAuthorizationModel int
+
+	maxOpenConns    int
+	maxIdleConns    int
+	connMaxIdleTime time.Duration
+	connMaxLifetime time.Duration
 }
 
 type MySQLOption func(*MySQL)
 
-func WithMaxTuplesInWrite(maxTuples int) MySQLOption {
+func WithTracer(t trace.Tracer) MySQLOption {
```

Comment:
Weird diff. Just moved this a bit so that MySQL and Postgres look the same and have these option functions in order.